scout/adapter/mongo/query.py

```diff
@@ -247,9 +246,19 @@ def build_query(
                 continue
 
             if criterion == "chrom" and query.get("chrom"):  # filter by coordinates
+                query_chrom = query.get("chrom")
```

Comment:
What about unselecting "All" if any specific chromosome is selected in the multiselect? Because the current code allows you to do this:

![image](https://user-images.githubusercontent.com/28093618/146906457-f7094df6-4d03-4c8c-a331-bb89150e4c8a.png)


Which is not returning interesting results, if the user meant to just select chromosome 1 and 2 for instance

Reply:
Sure! I gave it a shot, but the logic was so complex I decided to unwrap the functions a bit, using the event handler a bit more. Prototyped on the sv page now. I still haven't found a quick way to update the select "summary" when any option status changes. I guess something that triggers on input and summarises...

Reply:
Its sort of ok now methinks. The only thing is the immediacy of the values changing behind the scenes when All is selected. If one has the dropdown open already, it does not change to reflect the current values, although e.g. the summary on dropdown control does.

Reply:
I'm testing the latest version. The simplest situation is when you want to select a single chromosome, for instance chr3. If you select it, you find yourself in this situation and you have to manually remove "All" otherwise the research will return all chroms:

![image](https://user-images.githubusercontent.com/28093618/151942896-390deefe-fe0f-44ee-8d76-31125e824aee.png)

This behavior is a bit cumbersome since I think most searches would include one chromosome only..

I understand the difficulty of the task, because what you want is:
- to deselect "All" when any single chromosome is selected
- to deselect all any single chromosomes when "All" is the last selected item of the select.

I think jquery is more useful than javascript to achieve the above. Check for instance the snippet provided here: 
https://stackoverflow.com/questions/39329332/bootstrap-multiselect-how-to-uncheck-and-deselect-multiselect-options-using-jqu